### PR TITLE
update tutorials

### PR DESCRIPTION
**Pull request by AWehenkel**
Some clarifications and cleanings.

### CODE REVIEW

File: tutorials/lecture2-logistic-regression.ipynb [hunk 1]
```diff
@@ -192,7 +191,7 @@
     "# Loss and Optimizer\n",
     "# Softmax is internally computed.\n",
     "# Set parameters to be updated.\n",
-    "criterion = nn.NLLLoss()  \n",
+    "criterion = nn.CrossEntropyLoss()  \n",
```
Comment (glouppe) [inline tutorials/lecture2-logistic-regression.ipynb]: Given your change above, I am not sure this is correct, since `CrossEntropyLoss` expects logits.
